app/jobs/catalogue_check_job.rb

```diff
@@ -0,0 +1,76 @@
+class CatalogueCheckWorker < ActiveJob::Base
+  queue_as :default
+
+  require 'capybara'
+  require 'capybara/poltergeist'
+  require_relative '../../app/jobs/scrape_job.rb'
+  require_relative '../../app/jobs/delete_job.rb'
+
+  include Capybara::DSL
+  Capybara.register_driver :poltergeist do |app|
+    Capybara::Poltergeist::Driver.new(app, timeout: 60, js_errors: false)
+  end
+  Capybara.default_driver = :poltergeist
+
+  def perform
+
+    #setting the initial variable
+    current_catalogue = Hash.new(0)
+    compare_catalogue_array = []
+    new_catalogue = []
+    catalogue_delete = []
+    region_array = ["Melbourne, 3000", "Sydney, 2000"]

```

Comment:
`regions` would be a more concise name than `region_array`. Avoid putting data type in the variable name most of the time.